test/src/unit-capi-group.cc

```diff
@@ -54,6 +54,12 @@
 
 using namespace tiledb::test;
 
+#ifndef TILEDB_TESTS_ENABLE_REST
+constexpr bool rest_tests = false;
+#else
+constexpr bool rest_tests = true;
+#endif
```

Comment:
Just a heads up, we merged TileDB-Inc/TileDB-Internal#17 so we should probably switch to using the `VFSTestSetup` approach added in #4678 once that is merged. The `TILEDB_TESTS_ENABLE_REST` build option will be removed.

Reply:
That's right, we need to adapt the test. We don't need to go all the way to use `VFSTestSetup` here though, let me just write a suggestion on how to do minimal changes on that test.

Reply:
In fact it should be as simple as removing completely lines 57-61 and instead of `rest_tests` boolean use `fs_vec_[0]->is_rest()` to check if we are on REST CI. Let me know if that doesn't work.

Reply:
Was just working on this fix so thanks for the update. I'll try that and let you know if I have trouble